.github/workflows/template_build.yml

```diff
@@ -44,36 +47,28 @@ jobs:
           - 5432:5432
     steps:
       - uses: actions/checkout@v3
-      - name: Check inputs
+      - name: Health check
         run: |
           python --version
           echo "Hello build note: ${{ inputs.build_note }}"
           echo "Hello logging level: ${{ inputs.logging_level }} (directly from action inputs)"
           echo "Hello logging level: $LOGGING_LEVEL (from envar from action inputs)"
           echo "Running python.run_logging ..."
-          python -m python.run_logging
-
-      - name: Setup build environment
-        working-directory: ./
-        run: ./bash/docker_container_setup.sh
+          python -m products.template.python.run_logging
 
-      - name: 1) Data loading
+      - name: Load
         run: |
-          echo "PLACEHOLDER running Data loading"
+          python3 -m products.template.build_scripts.load
```

Comment:
I think we'd probably want default behavior to be running these from the product folder itself (in this case `./products/template`. Otherwise your imports will be janky when you run as a module, e.g. you'd have to do something like `from products.template.build_scripts import load` or only use relative imports. 

I think. Python imports are confusing.